solidity/random-beacon/scripts/init-contracts.ts

```diff
@@ -0,0 +1,89 @@
+/* eslint-disable @typescript-eslint/no-unused-expressions, import/no-extraneous-dependencies */
+import { ethers, helpers } from "hardhat"
+
+import type { T, TokenStaking, RandomBeacon } from "../typechain"
+
+const { to1e18 } = helpers.number
+
+const BLOCK_TIME = 5000 // in ms
+
+async function getAccounts() {
+  const provider = new ethers.providers.JsonRpcProvider(process.env.NETWORK)
+  return provider.listAccounts()
+}
+
+async function init() {
```

Comment:
We could rename the script file from `init-contracts.ts` to `stake.ts` and rename this function, as we're not initializing the contracts but staking.

```suggestion
async function stake() {
```

Reply:
Yea, I see why it might be confusing, but I don't think `stake.ts` & `stake()` fit perfectly here because `stake.ts` suggests that we only execute a `stake` part here, which is not true. We also do authorization and [regestring an operator](https://github.com/keep-network/keep-core/pull/3026#discussion_r901686383). Same with `stake()` which suggests we execute only a `stake()` function. I think renaming to `staking.ts` and `setup()` are more general namings for functionalities around "staking".